data/json/recipes/recipe_others.json

```
@@ -3094,6 +3094,58 @@
     "tools": [ [ [ "welder", 100 ], [ "welder_crude", 150 ], [ "toolset", 150 ], [ "soldering_iron", 150 ] ] ],
     "components": [ [ [ "steel_chunk", 6 ], [ "scrap", 18 ] ], [ [ "pipe", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "heavy_air_cylinder",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_MATERIALS",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "200 m",
+    "book_learn": [ [ "textbook_fabrication", 4 ], [ "manual_fabrication", 4 ] ],
+    "using": [ [ "welding_standard", 25 ] ],
+    "qualities": [ { "id": "HAMMER", "level": 2 } ],
```

```suggestion
    "using": [ [ "welding_standard", 25 ] ],
    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "GLARE", "level": 2 } ],
```
Add to the other ones as well, b/c welding is bad for eyes. 😛